web/cobrands/peterborough/assets.js

```diff
@@ -163,42 +189,62 @@ fixmystreet.assets.add(light_defaults, {
     asset_item_message: ''
 });
 
-var bin_defaults = $.extend(true, {}, defaults, {
-    class: OpenLayers.Layer.PeterboroughVectorAsset,
-    select_action: true,
-    actions: {
-        asset_found: fixmystreet.message_controller.asset_found,
-        asset_not_found: fixmystreet.message_controller.asset_not_found
-    },
-    attributes: {
-        asset_details: function() {
-            var a = this.attributes;
-            return a.Reference + ", " + a.Location;
-        },
-        central_asset_id: 'OBJECTID'
-    },
-    asset_id_field: 'OBJECTID',
-    asset_type: 'spot'
-});
+var url_base = 'https://tilma.mysociety.org/resource-proxy/proxy.php?https://peterborough.assets/';
 
-fixmystreet.assets.add(bin_defaults, {
+// PCC Property Combined
+fixmystreet.assets.add(arcgis_defaults, {
     http_options: {
+      url: url_base + '2/query?',
       params: {
-        TYPENAME: 'LitterBins'
+        outFields: '',
       }
     },
-    asset_category: 'Litter bin',
-    asset_item: 'litter bin'
+    stylemap: fixmystreet.assets.stylemap_invisible,
+    asset_category: ['General fly tipping', 'Hazardous fly tipping', 'Offensive graffiti', 'Non offensive graffiti'  ],
+    non_interactive: true,
+    road: true,
+    asset_item: 'road',
+    asset_type: 'road',
+    actions: {
+        found: function(layer) {
+            $("#js-roads-responsibility").addClass("hidden");
+        },
+        not_found: function() {
+            for ( var i = 0; i < fixmystreet.assets.layers.length; i++ ) {
+                var layer = fixmystreet.assets.layers[i];
+                if ( layer.fixmystreet.name == 'Adopted Highways' && layer.selected_feature ) {
+                    $("#js-roads-responsibility").addClass("hidden");
+                    return;
+                }
+            }
+            $("#js-roads-responsibility").removeClass("hidden");
+            $("#js-roads-responsibility .js-responsibility-message").addClass("hidden");
+            $('#js-environment-message').removeClass('hidden');
+        },
+    }
 });
 
-fixmystreet.assets.add(bin_defaults, {
+// PCC Property Leased Out NOT Responsible
+fixmystreet.assets.add(arcgis_defaults, {
     http_options: {
+      url: url_base + '3/query?',
       params: {
-        TYPENAME: 'DogBins'
+        outFields: '',
       }
     },
-    asset_category: 'Dog bin',
-    asset_item: 'dog waste bin'
+    stylemap: fixmystreet.assets.stylemap_invisible,
+    asset_category: ['General fly tipping', 'Hazardous fly tipping', 'Offensive graffiti', 'Non offensive graffiti'  ],
```

Comment:
Would be good to factor this together with the `PCC Property Combined` one above, if only to make it easier to update these lists of categories in the future.